test/Chainweb/Test/Pact/ModuleCacheOnRestart.hs

```diff
@@ -76,29 +76,28 @@ tests rdb =
       withMVarResource mempty $ \iom ->
       withEmptyMVarResource $ \rewindDataM ->
       withTestBlockDbTest testVer rdb $ \bdbio ->
-      withTemporaryDir $ \dir ->
+      withTempSQLiteResource $ \ioSqlEnv ->
```

Comment:
Instead of reopening the database file on for each test, we open the database once and pass the `SQLiteEnv` to the tests.

In any case, the tests must be executed sequentially, which isn't changed by this PR.